rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/V2BakeryController.java

```diff
@@ -1,24 +1,31 @@
 package com.netflix.spinnaker.rosco.controllers;
 
 import com.netflix.spinnaker.kork.artifacts.model.Artifact;
-import com.netflix.spinnaker.rosco.manifests.BakeManifestRequest;
-import com.netflix.spinnaker.rosco.manifests.helm.HelmBakeManifestRequest;
-import com.netflix.spinnaker.rosco.manifests.helm.HelmBakeManifestService;
+import com.netflix.spinnaker.rosco.manifests.BakeManifestService;
 import groovy.util.logging.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
I don't think we want `*` here. It's best to be explicit about imports.

Reply:
fixed